invenio/base/scripts/apache.py

```diff
@@ -17,195 +17,508 @@
 # along with Invenio; if not, write to the Free Software Foundation, Inc.,
 # 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
 
+"""Apache configuration generator.
+
+This script relies on the following variables:
+
+    Note: The variables are followed by their defaults.
```

Comment:
see an example how to document config variables: http://invenio.readthedocs.org/en/latest/ext/arxiv.html?highlight=config#configuration-settings, https://github.com/inveniosoftware/invenio/blob/pu/invenio/ext/arxiv/__init__.py#L29-L44